e2e/specs/menu_bar/file_menu.test.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+'use strict';
+
+const fs = require('fs');
+
```

Comment:
nit : can we remove this empty line since both of them are external module imports?

Reply:
sorry my bad this changed caused lint error. but i wonder why https://app.circleci.com/pipelines/github/mattermost/desktop/3671/workflows/d7848497-e726-45d7-a644-db615d92b331/jobs/16296?invite=true#step-106-13

Reply:
i reverted the change